server/src/test/java/org/spine3/server/aggregate/AggregateRootShould.java

```diff
@@ -21,51 +21,71 @@
 package org.spine3.server.aggregate;
 
 import com.google.protobuf.Message;
+import io.grpc.stub.StreamObserver;
 import org.junit.Before;
 import org.junit.Test;
+import org.spine3.base.Command;
 import org.spine3.base.CommandContext;
+import org.spine3.base.Commands;
+import org.spine3.base.Response;
 import org.spine3.server.BoundedContext;
 import org.spine3.server.command.Assign;
-import org.spine3.test.aggregate.Project;
+import org.spine3.server.command.CommandBus;
+import org.spine3.test.aggregate.ProjectDefinition;
 import org.spine3.test.aggregate.ProjectId;
-import org.spine3.test.aggregate.command.AddTask;
+import org.spine3.test.aggregate.ProjectLifecycle;
+import org.spine3.test.aggregate.Status;
 import org.spine3.test.aggregate.command.CreateProject;
 import org.spine3.test.aggregate.command.StartProject;
 import org.spine3.test.aggregate.event.ProjectCreated;
 import org.spine3.test.aggregate.event.ProjectStarted;
-import org.spine3.test.aggregate.event.TaskAdded;
-import org.spine3.testdata.Sample;
 
+import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.atMost;
 import static org.mockito.Mockito.spy;
 import static org.mockito.Mockito.verify;
+import static org.spine3.base.Identifiers.newUuid;
+import static org.spine3.testdata.TestCommandContextFactory.createCommandContext;
 
 public class AggregateRootShould {
 
-    private AggregateRoot aggregateRoot;
+    private static final StreamObserver<Response> RESPONSE_OBSERVER = new MockStreamObserver();
```

Comment:
> RESPONSE_OBSERVER
Please read [Constant Names](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) section in Google Java Code Style.

Fix the variable name name accordingly.